scripts/release/build/rpm/sign.sh

```diff
@@ -5,44 +5,44 @@
 set -ex
 
 export HOME=/root
-export GOPATH=${HOME}/go
-export PATH=${GOPATH}/bin:/usr/local/go/bin:${PATH}
+export GOPATH="$HOME/go"
+export PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
 
-cd ${HOME} && tar jxf /root/subhome/gnupg*.tar.bz2
+cd "$HOME" && tar jxf /root/subhome/gnupg*.tar.bz2
 
-export PATH="${HOME}/gnupg2/bin:${PATH}"
-export LD_LIBRARY_PATH=${HOME}/gnupg2/lib
+export PATH="$HOME/gnupg2/bin:$PATH"
```

Comment:
Why are you removing the curly braces?

Reply:
Because they're unnecessary in these situtations.